core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -214,6 +214,13 @@ private static String toSql(RelNode root, SqlDialect dialect,
         .getSql();
   }
 
+  @Test void testGroupByBoolean() {
+    String query = "select avg(\"salary\") from \"employee\" group by true";
+    String expected = "SELECT AVG(\"employee\".\"salary\")\nFROM \"foodmart\".\"employee\",\n"
+        + "(VALUES (TRUE)) AS \"t\" (\"T\")\nGROUP BY \"t\".\"T\"";
```

Comment:
Did you verify that the generated query works in `Redshift? This is slightly different from the query discussed in the JIRA so let's ensure it works:
```
select avg(sal)
 from emp, (select true x) dummy
 group by dummy.x;
```